SAMPLE: Ethnography sample data from Cassandre.

julien-millour opened the pull request:
We, Romain PHILIPPE, Corentin PHILIPPE, Antoine ROCHE, Majed HLAIHEL and Julien MILLOUR, hereby grant to Hyperglosae maintainers the right to publish our contribution under the terms of any licenses the Free Software Foundation classifies as Free Software Licenses.

christophe-lejeune commented:
As mentioned during our interview, the date in this sample should be corrected ( `dc_issued`: `2012`).

julien-millour commented:
> As mentioned during our interview, the date in this sample should be corrected ( `dc_issued`: `2012`).

@christophe-lejeune Are you alright with the licence that we chose in the last sample (CC by 4.0) or do should we use another one?

christophe-lejeune commented:
I agree with the proposed license. 

May I suggest that this sample also includes the coding memos I originally provided with these theoretical / operational fields memos ?

christophe-lejeune commented:
Perhaps "Ethnography" would be more inclusive / broad / general than "Ethnomethodology"...

julien-millour commented:
> May I suggest that this sample also includes the coding memos I originally provided with these theoretical / operational fields memos ?

@christophe-lejeune Could you specificy what the coding memos are? We're not sure of what you're referring.

Would the memos be what's displayed on the right here :  
![image](https://user-images.githubusercontent.com/126160064/231970827-c0aaffba-cbff-42ec-864e-f64f01bba716.png)


Or here: 
![image](https://user-images.githubusercontent.com/126160064/231970749-c5c5f376-3d93-40b1-9380-27f275a43bdd.png)

christophe-lejeune commented:
The original dataset was transmitted to @benel . He then gave it to you. Perhaps he has good reasons to exclude the coding memos. I suggest you discuss this issue with him ;-)

benel commented on samples/eth_inrapA.json:
```diff
@@ -0,0 +1,9 @@
+  "dc_title": "Restaurer la vapeur",
+  "dc_creator": "Christophe Lejeune",
+  "text": "Comment se coordonne la restauration de locomotives à vapeur ? Comment les bénévoles acquièrent-ils collectivement les compétences techniques et les habiletés sociales permettant de faire rouler chaque semaine des locomotives à vapeur ? Comment se construit l'identité de celui ou celle qui est prêt-e à passer ses week-ends à poncer des essieux de locomotives ? Comment cet investissemen ...
+  "couchapp": {}
```
This is an artefact from pushing the document with the command.  This shouldn't be in the original sample.

benel commented on samples/eth_inrapB.json:
```diff
@@ -0,0 +1,15 @@
+  "dc_title": "Se rendre à l'atelier",
+  "dc_creator": "Christophe Lejeune",
+  "text": "\n * Dans l'atelier / en ligne / au musée / sur le faisceau \n* Chacun de son côté / en sous groupe / en grand groupe \n* Apprendre / transmettre / coacher \n* Parrainage / mentorat / compagnonnage",
+  "couchapp": {}
```
This is an artefact from pushing the document with the command.  This shouldn't be in the original sample.

benel commented on samples/eth_inrapC.json:
```diff
@@ -0,0 +1,15 @@
+  "dc_title": "Rencontrons un président",
+  "text": "* Origine, d'où ? \n* Train électrique \n* Milieu des cheminots\n* En faire un travail \n* Trains touritiques\n",
+  "dc_creator": "Christophe Lejeune",
+  "couchapp": {}
```
This is an artefact from pushing the document with the command.  This shouldn't be in the original sample.

benel commented on samples/eth_inrapD.json:
```diff
@@ -0,0 +1,16 @@
+  "dc_language": "fra",
+  "dc_title": "Treignes, le 8 septembre 2012",
+  "dc_creator": "Christophe Lejeune",
+  "couchapp": {}
```
This is an artefact from pushing the document with the command.  This shouldn't be in the original sample.

benel reviewed (changes requested):
Thank you for this interesting sample. To be merged you should fix the following:
- [ ] `couchapp` is an artefact of the tool you used, it shouldn't be in the original data,
- [ ] you told me the coding memo was done, don't forget to add it to the commit,
- [ ] your sample has nothing to do with INRAP hence I don't understand the naming convention of your files,
- [ ] your stakeholder told you that, for this sample, he preferred the term "Ethnology" to "Ethnomethodology", therefore you should rename your commit and your files accordingly.

benel commented:
@Hypertopic/ethnomethodologie 

If I remember rightly, you fixed the problems last week on your local computer. 
Could you please update your branch so that I can merge your samples?

Regards.

julien-millour commented:
@benel I updated the files and the commit with the correct name and removed the data that shouldn't appear in the sample. However, I had issues when retesting the coding memo and I wanted to show them to you tomorrow. Would that be possible?

benel commented on samples/ethnography05.json:
```diff
@@ -0,0 +1,15 @@
+      "object": "6b56ee657c870dfacd34e9ae4e0643dd"
+    }
+  ],
+  "text": "{1} \n {2} Se socialiser\nDonner son avis sur une tâche en cours\n {3} \n {4} Se concerter (lors de la pause du midi)",
```
With this version, you didn't "grounded" the code precisely in the text. Please note that: 
- to get a precise anchor you need both the block identifier and the quoted text,
- as stated in [documentation](https://github.com/Hypertopic/HyperGlosae/blob/6a1b4a56dea8d3d0fb2a3de3e49d972c503b3848/samples/inrap_D0.json#L12), the quoted text should use [Markdown definition lists](https://www.markdownguide.org/extended-syntax/#definition-lists).

Please test with the following `text`:

```json
"{2} Se socialiser\n: plusieurs personnes se présentent à moi. Ayant identifié que je suis nouveau, elles me souhaitent la bienvenue\n\nDonner son avis sur une tâche en cours\n: Un autre m'indique que j'aurais intérêt à commencer par les parties hautes de la locomotive, plutôt que des côtés, car celles-ci deviendront vite très chaudes (je risque de me bruler si j'attends pour les laver){4} Se concerter (lors de la pause de midi)\n: Luc et Leo exposent les résultats des essais. Les différentes personnes présentes discutent des solutions possibles"
```

benel reviewed (changes requested):
Thank you for your contribution.

To be integrated your sample however should be fixed.
The "codes" (annotations) must comply with the format specified in the documentation.
